.github/workflows/lint-test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       PRE_COMMIT_HOME: ${{ github.workspace }}/.cache/pre-commit-cache
 
       AUTH_TOKEN: ci-token
-      DATABASE_URL: postgres://pysite:pysite@postgres:5432/pysite
+      TESTING: true
```

Comment:
We shouldn't need this or any of the test patching, we can just use a local database for local testing and the actions, we already have a database service, see:

https://github.com/python-discord/api/blob/6df16aea2868c39a60b5e42189e507be6ebbfc35/.github/workflows/lint-test.yml#L29-L44

Reply:
It was discussed on discord, that we should try not to use a database for testing so that we can implement further features in the future.